andino_apps/CHANGELOG.rst

```diff
@@ -0,0 +1,9 @@
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Changelog for package andino_apps
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+0.1.0 (2024-04-10)
+------------------
+* Initial version of the package
+* Moved launch file to execute Gazebo classic simulation + Nav2 from `andino_navigation` (`#228 <https://github.com/Ekumen-OS/andino/issues/228>`_)
+* Contributors: Jesús Silva
```

Comment:
Probably can remove, git takes care of it via blame :)

Reply:
Understood!

Reply:
I checked the others and it seems it's the standard practice. Sorry about this comment. No action required.